### PR TITLE
Use `mamba-org/setup-micromamba`

### PR DESCRIPTION
**Pull request by mattwthompson**
The same changes to the deploy script might be useful

### CODE REVIEW

File: .github/workflows/ci-integration.yml [hunk 1]
```diff
@@ -33,15 +33,11 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - uses: conda-incubator/setup-miniconda@v2
+      - uses: mamba-org/setup-micromamba
```
Comment (dotsdl) [inline .github/workflows/ci-integration.yml:36]:
```suggestion
      - uses: mamba-org/setup-micromamba@v1
```

**Comment by dotsdl** [comment]
@mattwthompson seeing the following; any insights?

```
Run mamba info -a
/home/runner/work/_temp/4bd3f59a-5ac3-46bf-9b6a-cbcd49aac3b2.sh: line 1: mamba: command not found
/home/runner/work/_temp/4bd3f59a-5ac3-46bf-9b6a-cbcd49aac3b2.sh: line 2: mamba: command not found
Error: Process completed with exit code 127.
```

**Comment by mattwthompson** [comment]
That should just be that `mamba` doesn't exist in this setup and isn't aliased (`conda`, for some reason, is aliased in that action)

**Comment by dotsdl** [comment]
Oh awesome, thank you so much @mattwthompson! You saved me a lot of flailing about here!

The `mambaforge`/`miniforge`/`conda`/`miniconda`... landscape has been hard to keep up with lately, so thanks for helping us here!

**Pull request merged**